inc/saihostintf.h

```diff
@@ -558,8 +558,10 @@ typedef enum _sai_hostif_packet_attr_t
     /** packet transmit type [sai_hostif_tx_type_t]. (MANDATORY_ON_SEND) */
     SAI_HOSTIF_PACKET_ATTR_TX_TYPE,
 
-    /** Egress port or LAG [sai_object_id_t] (for send-only).
-     * (MANDATORY_ON_SEND when SAI_HOSTIF_PACKET_TX_TYPE == SAI_HOSTIF_TX_TYPE_PIPELINE_BYPASS) */
+    /** Egress port or LAG [sai_object_id_t].
```

Comment:
It should be filled with port and application can lookup the LAG Id from port when needed. If a specific SDK/ASIC provides only the LAG for some reason, that can be filled in the attribute in that case. I have just removed the comment that defines this attribute "for send-only" case.